package.json

```diff
@@ -98,6 +98,8 @@
         "node-sass-tilde-importer": "^1.0.2",
         "npm-run-all": "^4.1.5",
         "parcel-bundler": "^1.12.4",
+        "patch-package": "^6.2.2",
+        "postinstall-postinstall": "^2.1.0",
         "prettier": "2.0.5",
```

Comment:
Stod det noe i patch-package docen om å låse pakkene til en låst versjon

Reply:
I mitt hodet så høres det lurt ut siden vi patcher kode direkte i node_modules.

Reply:
Kunne ikke se noe spesifikt om det. Vi har jo for så vidt låst pakkene nå pga den ene bugen, så det skal gå fint. Men tenkte jeg skulle undersøke litt mer uansett!

Reply:
Tipper patch-package klarer ser om diffen er aneledes og dermed gi en eller annen form for tilbakemelding, så går sikkert fint. Men tror det er greit å låse for å ha kontroll bakover også.

Reply:
Ja. Men tenkte uansett å oppgradere til nyeste versjoner av `core-components` siden det ikke lenger er konflikt. Så kan vi heller låse versjonene der.

Reply:
Oppgraderte `core-toggle` til 3.0.8 nå. Kan bekrefte at det kom tilbakemelding 😊 I dette tilfellet kunne den patche likevel, så den gjorde det automatisk. Har likevel oppdatert patchen så den gjelder riktig versjon. Nå er både `core-toggle` og `core-datepicker` låst til 3.0.8 inntil videre. Kommer vel neppe noen nye versjoner før vi kan droppe patchene uansett 😅